### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Edit Configuration popup_Keyboard: Scroll bar is not accessible using keyboard.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3601]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS2.1.1; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS 2.1.1

**NOTE:
Same issue is observed in “Spark Console” scenario also.
(Run --> Edit Configurations-->Apache Spark on HDInsight > [Spark on HDInsight] myApp)**

**Repro Steps:**
1.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA.
3.Tab till Create New Project button and press enter
4.Tab till Azure Spark/HDInsight-->Spark Project Scala
5.Tab till Build Tools-->Maven
6.Tab till Next button and press enter
7.Tab till finish button and press enter
8.Navigate to project name-->src-->main-->scala-->project name and open the context menu using keyboard key
9.Tab till Submit Spark Application and press enter.
10.Tab till “Referenced files” field.
11.Verify whether Scroll bar is accessible using keyboard or not.

**Actual Result:**
Scroll bar is not accessible using keyboard. Due to this, the complete data present on the page is not visible to user.  

**Expected Result:**
Scroll bar should be accessible using keyboard, Inorder to avoid loss of visibility of data.

**User Impact:**
Keyboard users will face difficulty in accessing and knowing the complete information present if scroll bar is not accessible using keyboard.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Bug_3_Scrollbar is not accessible using keyboard in edit configuration popup.pptx](https://github.com/microsoft/azure-tools-for-java/files/3642118/Bug_3_Scrollbar.is.not.accessible.using.keyboard.in.edit.configuration.popup.pptx)

**Comment by t-rufang** [comment]
Note: scroll bar is not accessible.
![image](https://user-images.githubusercontent.com/32627233/65659309-cac50280-e05d-11e9-91b7-bae6945bcf0e.png)

**Comment by t-rufang** [comment]
Fixed in #3601. Now User can use `Up` and `Down` arrow in the keyboard to see up and down view in the panel.

**Comment by v-lasiva** [comment]
Regressed the issue in the updated version of IntelliJ having,
Version: 2019.2.2

Scroll Bar is now accessible using keyboard.
As the issue is resolved, Hence closing the bug.
Added attachment for reference.

[Bug_Scroll bar is now accessible_Issue fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3846566/Bug_Scroll.bar.is.now.accessible_Issue.fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749704
